docs-src/Makefile

```diff
@@ -1,10 +1,27 @@
+L0_VERSION:=$(shell git describe --tags)
+DOWNLOAD_LINK:=| v0.10.4 | \
+					[macOS](https://github.com/quintilesims/layer0/releases/download/v0.10.4/macOS.zip) | \
+					[Linux](https://github.com/quintilesims/layer0/releases/download/v0.10.4/Linux.zip) | \
+					[Windows](https://github.com/quintilesims/layer0/releases/download/v0.10.4/Windows.zip
+
 deps:
 	pip install mkdocs
 	pip install -Iv mkdocs-material==2.3.0
 
 build:
 	mkdocs build --clean
 
+release:
+	# Remove v from Version
+	$(eval L0_VERSION := $(shell echo $(L0_VERSION) | cut -c 2-))
```

Comment:
For `README.md`, I think using `sed` along with stripping the `v` prefix is fine. 

For the rest of the files, I worry that using the stripped prefix could cause inconsistencies. 
I think we should consider using a tool like [expenv](https://github.com/blang/expenv) which replaces environment variables in template files. That way, we could include templated files, e.g. `docs-src/index.md.template` that contained:
```
...
## Download
| Download **$L0_VERSION** | |
...
```

and in this `Makefile`, we would have:
```
expenv -f docs/index.md.template > docs/index.md
```

For the problem of adding a new table entry, we could add a new template, reverse the order of our table, and use the append `>>` function: 
```
expenv -f docs/release_table.template >> docs/releases.md
```

Reply:
Okay sweet, I was wondering about using a template file but wasn't sure if the added resources would be wanted. I'll switch to this.